sema4ai/vscode-client/src/viewsRobots.ts

```diff
@@ -275,6 +275,15 @@ export class RobotsTreeDataProvider implements vscode.TreeDataProvider<RobotEntr
                         "parent": element,
                         "tooltip": "Publishes the Action Package to the Sema4.ai Desktop application",
                     },
+                    {
+                        "label": "Build Package to Workspace",
```

Comment:
Hmmm...
I think we should have a more suggestive copy here. 🤔 
Maybe:
```suggestion
                        "label": "Create Action Package",
```

Or:
```suggestion
                        "label": "Export Action Package to Workspace",
```

What do you think?

Reply:
Yeah that sounds good, build may be too technical.